apim-lab/4-policies-expressions/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Policies Expressions
```

Comment:
Please revert all instances of `Policies Expressions` to `Policy Expressions` as this is a singular term:

https://learn.microsoft.com/en-us/azure/api-management/api-management-policy-expressions